pages/index.js

```diff
@@ -237,128 +237,150 @@ const FooterContent = styled(Content)`
-        <ArtsyLogo />
-      </CompanyLogo>
-
-      <CompanyLogo height="1.75rem">
-        <AtlassianLogo />
-      </CompanyLogo>
-
-      <CompanyLogo>
-        <Auth0Logo />
-      </CompanyLogo>
-
-      <CompanyLogo>
-        <BBCLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="-0.2rem" height="1.75rem">
-        <BloombergLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="0.75rem" height="3rem">
-        <BlueTomatoLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="-0.2rem">
-        <CasperLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="0.3rem" height="2rem" src="/static/coinbase-logo.svg">
-        <CoinbaseLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="-0.2rem">
-        <DoorDashLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="0.1rem" height="2.5rem">
-        <EuroVisionLogo />
-      </CompanyLogo>
-
-      <CompanyLogo src="/static/huffpost-logo.svg">
-        <HuffpostLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="-0.15rem" height="1.5rem">
-        <IdeoLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="0.1rem" height="2.25rem">
-        <InVisionLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="0.3rem" height="2.75rem">
-        <KiwiComLogo />
-      </CompanyLogo>
-
-      <CompanyLogo>
-        <LegoLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="1rem" height="4rem">
-        <MagicLeapLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="-0.15rem" height="1.3rem">
-        <MoleskineLogo />
-      </CompanyLogo>
-
-      <CompanyLogo>
-        <PatreonLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="0.15rem">
-        <RedditLogo />
-      </CompanyLogo>
-
-      <CompanyLogo>
-        <SesameLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="0.4rem" height="3rem">
-        <ShpockLogo />
-      </CompanyLogo>
-
-      <CompanyLogo>
-        <TargetLogo />
-      </CompanyLogo>
-
-      <CompanyLogo height="1.7rem">
-        <TicketmasterLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="0.2rem">
-        <TinderLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="0.15rem" height="2.35rem">
-        <TypeformLogo />
-      </CompanyLogo>
-
-      <CompanyLogo bottom="0.3rem" height="2.1rem">
-        <VimeoLogo />
-      </CompanyLogo>
-
-      <CompanyLogo height="1.8rem">
-        <VogueLogo />
-      </CompanyLogo>
+const SortedLogos = ({ children }) => {
+  // sorting logic: the more popular a website, the higher it gets listed
+  const similarWebGlobalRankSortedLogos = React.Children.toArray(children).sort(
+    (a, b) => a.props.similarWebGlobalRank - b.props.similarWebGlobalRank
+  );
 
-      <CompanyLogo bottom="0.7rem" height="2.5rem">
-        <XingLogo />
-      </CompanyLogo>
-    </>
+  return (
+    <UsersWrapper>
+      {similarWebGlobalRankSortedLogos}
+      {similarWebGlobalRankSortedLogos}
```

Comment:
Is there a reason they are rendered twice?

Reply:
That's for the animation but previously the logos were in a component that was rendered twice, so this is correct 👌 there might be a smarter way to make this work though

Reply:
jap, saw it was rendered twice before, so I just kept it for now.